src/LibraryManager.Vsix/Commands/CommandTable/VSCommandTable.en.vsct

```diff
@@ -49,6 +49,7 @@
         <CommandFlag>TextChanges</CommandFlag>
         <Strings>
           <ButtonText>Enable Restore Client-Side Libraries on Build...</ButtonText>
```

Comment:
>... [](start = 67, length = 3)

Sorry, I'm out of the loop here. I assume this brings up a dialog?

Reply:
Clarified that this feature DOES require ellipses.
Checked that the counterpart - Disable - does NOT require ellipses.
Modified string in Text.resx to add ellipses.